samples/constitutions/default/actions.js

```diff
@@ -1577,4 +1577,39 @@ const actions = new Map([
       function (args) {},
     ),
   ],
+  [
+    "cleanup_legacy_jwt_records",
+    new Action(
+      function (args) {
+        checkType(
+          args.ensure_new_records_exist,
+          "boolean?",
+          "ensure_new_records_exist",
+        );
+      },
+      function (args) {
+        if (
+          args.ensure_new_records_exist &&
+          ccf.kv["public:ccf.gov.jwt.public_signing_keys_metadata_v2"].size ===
+            0
+        ) {
+          throw new Error("No new JWT public signing keys records found");
+        }
+
+        const keys = ccf.kv["public:ccf.gov.jwt.public_signing_keys"];
+        if (keys !== undefined) {
+          keys.clear();
+        }
+        const metadata =
+          ccf.kv["public:ccf.gov.jwt.public_signing_keys_metadata"];
+        if (metadata !== undefined) {
+          metadata.clear();
+        }
+        const issuer = ccf.kv["public:ccf.gov.jwt.public_signing_key_issuer"];
+        if (issuer !== undefined) {
+          issuer.clear();
+        }
```

Comment:
```suggestion
        ccf.kv["public:ccf.gov.jwt.public_signing_keys"].clear();
        ccf.kv["public:ccf.gov.jwt.public_signing_keys_metadata"].clear();
        ccf.kv["public:ccf.gov.jwt.public_signing_key_issuer"].clear();
```

The table lookup will always succeed and return a handle, there's no need to check it for `undefined` (and doing so in a sample constitution action may mislead users).

Reply:
Confirmed that works, maybe I've glitched. Simplified the proposal.